recipes/mpir/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+chmod +x configure
+
+./configure --prefix=$PREFIX --enable-cxx
+
+make
+
+if [ "$(uname)" != "Darwin" ];
+then
+    make check
```

Comment:
Builds timeout because the package is built 3 times in the same build. Why is that?

https://travis-ci.org/conda-forge/staged-recipes/jobs/118898136#L265
https://travis-ci.org/conda-forge/staged-recipes/jobs/118898136#L4201
https://travis-ci.org/conda-forge/staged-recipes/jobs/118898136#L8144